.travis.yml

```diff
@@ -12,3 +12,28 @@ script:
   - node dist/node/loader.entry.js
 after_success:
   - npm run report-coverage
+branches:
+  only:
+    - master
+    - /^v[0-9]/
```

Comment:
Yeah, but it won't effectively change anything because it only applies to branches and tags in this repo (PRs are always built).

Mainly, it's the simplest way to avoid building (and then trying to deploy) tags that don't look like version numbers.